data_sources/datastore_data.py

```diff
@@ -9,6 +9,55 @@
 from models.call_history_model import CallHistory
 
 
+class DataStoreClient:
+    def __init__(self):
+        self.client = datastore.Client()
+
+    def generate_year_old_test_data(self):
+        i = 1
+        while i < 601:
+            task = datastore.Entity(self.client.key("CallHistory"))
+            task.update(
+                {
+                    "call_start_time": datetime(2022, 10, 20),
```

Comment:
Should this use today's date + 1 year?

Will this fail if we ran it in a few days?